PAWS/ECGRecordings/ECGRecording.swift

```diff
@@ -44,14 +45,22 @@ struct ECGRecording: View {
                     Text("Recorded \(symptoms.count) symptoms.")
                 }
             }
-                .padding()
+            .padding()
         }
-            .task {
-                guard let symptoms = try? await electrocardiogram.symptoms(from: HKHealthStore()) else {
-                    return
+        .task {
+            guard let symptoms = try? await electrocardiogram.symptoms(from: HKHealthStore()) else {
+                return
+            }
+            
+            self.symptoms = symptoms
+            
+            if !FeatureFlags.disableFirebase {
+                do {
+                    self.isUploaded = try await ecgModule.isUploaded(electrocardiogram)
+                } catch {
+                    await ecgModule.addECGMessage(for: electrocardiogram)
```

Comment:
If we add the reupload if needed flag here we won't need to display the notification from here as well, it should just do this from within the ECG module.